nss-day-cohort-22/company-website-part-deux-deceptive-eels
# Sean safety page

# to test 

1. make sure the homepage loads
2. make sure that the top right, top left, and bottom right images open up a modal box.

## To make it test your stuff

1. pull down and merge master to your local branch
1. make sure the proper image in utilities.js has a call to your function
1. make sure your function inserts inside the div with the id "bubble"
